BitcoinApp/Application/Model/Enums.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Enums.swift
+//  BitcoinApp
+//
+//  Created by Zhassulan Aimukhambetov on 12/5/19.
+//  Copyright © 2019 Zhassulan Aimukhambetov. All rights reserved.
+//
+
+import Foundation
+
+enum Currency: Int {
+    case KZT
+    case USD
+    case EUR
+    
+    var name: String {
+        switch self {
+        case .USD: return "USD"
+        case .EUR: return "EUR"
+        case .KZT: return "KZT"
+        }
+    }
+    
+    var text: String {
+        switch self {
+        case .USD: return "＄"
+        case .EUR: return "€"
+        case .KZT: return "₸"
+        }
+    }
+}
+
+enum Period: Int {
+    case week
+    case month
+    case year
+    
+    var interval: TimeInterval {
+        switch self {
+        case .week: return (-7 * 60 * 60 * 24)
```

Comment:
Never use manual calculation for time. Time measurement is not as simple as 24 hours in a day with 60 seconds in a minute. 
On Apple platforms, we can use `DateComponents` with `Calendar`